tests/e2e/controlplane/control_plane_test.go

```diff
@@ -57,8 +57,12 @@ var _ = Describe("Control Plane Installation", Ordered, func() {
 			extraArg = "--set=platform=openshift"
 		}
 
-		Expect(helm.Install("sail-operator", filepath.Join(project.RootDir, "chart"), "--namespace "+namespace, "--set=image="+image, extraArg)).
-			To(Succeed(), "Operator failed to be deployed")
+		if skipDeploy {
+			Success("Skipping operator installation")
```

Comment:
Hey, maybe add here also: `Skipping operator installation, SKIP_DEPLOY set to true` or `Skipping operator installation, the operator is already installed`. Or something more similar to this messaage that you already added: `Success("Skipping operator undeploy because it was deployed externally")`